src/verify.v2.integration.test.ts

```diff
@@ -660,92 +580,86 @@ describe("verify(integration)", () => {
     const results = await verify(documentRopstenRevokedWithDocumentStore, {
       network: "ropsten",
     });
-
```

Comment:
might need to take a quick look at the description of tests
should fail for OpenAttestationEthereumDocumentStoreRevoked => OpenAttestationEthereumDocumentStoreStatus